src/components/ButtonPanel.js

```diff
@@ -2,43 +2,50 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import Button from './Button';
 
-function ButtonPanel() {
+function ButtonPanel(props) {
+  // eslint-disable-next-line react/prop-types
```

Comment:
Do not disable react/prop-types rule. Please remove this line and add the necessary rule to clear this warning/error.